Sources/BraintreePayPalNativeCheckout/BTPayPalNativeRequest.swift

```diff
@@ -6,17 +6,164 @@ import BraintreeCore
 import BraintreePayPal
 #endif
 
-protocol BTPayPalNativeRequest {
-    var hermesPath: String { get }
-    var paymentType: BTPayPalPaymentType { get }
+@objcMembers public class BTPayPalNativeRequest: NSObject {
 
-    func parameters(with configuration: BTConfiguration) -> [AnyHashable: Any]
-}
+    // MARK: - Public Properties
+    // next_major_version: subclass BTPayPalRequest once BraintreePayPal is in Swift.
+
+    /// Optional: The line items for this transaction. It can include up to 249 line items.
+    public var lineItems: [BTPayPalLineItem]?
+
+    /// Defaults to false. When set to true, the shipping address selector will be displayed.
+    public var isShippingAddressRequired: Bool
+
+    /// Optional: The merchant name displayed inside of the PayPal flow; defaults to the company name on your Braintree account
+    public var displayName: String?
+
+    ///  Optional: A locale code to use for the transaction.
+    ///  - Note: Supported locales are:
+    ///
+    /// `da_DK`,
+    /// `de_DE`,
+    /// `en_AU`,
+    /// `en_GB`,
+    /// `en_US`,
+    /// `es_ES`,
+    /// `es_XC`,
+    /// `fr_CA`,
+    /// `fr_FR`,
+    /// `fr_XC`,
+    /// `id_ID`,
+    /// `it_IT`,
+    /// `ja_JP`,
+    /// `ko_KR`,
+    /// `nl_NL`,
+    /// `no_NO`,
+    /// `pl_PL`,
+    /// `pt_BR`,
+    /// `pt_PT`,
+    /// `ru_RU`,
+    /// `sv_SE`,
+    /// `th_TH`,
+    /// `tr_TR`,
+    /// `zh_CN`,
+    /// `zh_HK`,
+    /// `zh_TW`,
+    /// `zh_XC`.
+    public var localeCode: String?
+
+    /// Defaults to false. Set to true to enable user editing of the shipping address.
+    /// - Note: Only applies when `shippingAddressOverride` is set.
+    public var isShippingAddressEditable: Bool
+
+    /// Optional: A valid shipping address to be displayed in the transaction flow. An error will occur if this address is not valid.
+    public var shippingAddressOverride: BTPostalAddress?
+
+    /// Optional: A risk correlation ID created with Set Transaction Context on your server.
+    public var riskCorrelationID: String?
```

Comment:
Updated this to `riskCorrelationID` vs riskCorrelationId` to match our preferred naming. This _is_ a breaking change but we are in beta currently. We can revert this and leave to be updated in next major version if preferred, but not sure what our pattern is on breaking changes in a beta. Thoughts?

Reply:
On Android we broke stuff in beta when needed and we didn't get complaints from merchants. It's a good time to break stuff.

Reply:
Should the target branch for this PR be `next-major-version`?

Reply:
We had a merchant report this issue on master and were able to replicate when using cocoapods on master with other Obj-C modules. My question about beta was because PayPalNativeCheckout is still technically in beta/limited release. Since the bug can be fixed on master, we should fix it there then pull the changes into next major version. 

Not sure if that changes your comment about breaking changes in beta. This would be a breaking change on master to the `BraintreePayPalNativeCheckout` module which is in beta... if that makes sense.

Reply:
Oh yeah I see now. It's probably best to defer the renaming until the beta release in that case to avoid introducing a breaking change.

Reply:
To clarify `BraintreePayPalNativeCheckout` module has already been released. We marked it as a "BETA" module in our docs and CHANGELOG.

Given that we've clearly documented it as a BETA module, I think we can make breaking changes to it (as long as they're called out in the CHANGELOG, of course).